src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The truck is working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("The truck stopped working");
```

Comment:
Similarly, the stopWork method should have a more informative message. Instead of "The truck stopped working", you could use "Truck has stopped working" to clearly indicate the type of machine.